src/app/conf/2024/schedule/[id]/page.tsx

```diff
@@ -100,111 +100,106 @@ export default function SessionPage({ params }: SessionProps) {
     videos.map(e => e.title),
   ).bestMatch
 
+  const videoId = videos.find(e => e.title === recordingTitle.target)?.id
+
+  if (!videoId) {
+    throw new Error(`Video "${recordingTitle.target}" not found`)
+  }
+
   return (
     <div className="bg-[#f4f6f8]">
       <div className="container">
         <div className="py-10">
           <section className="text-[#333333] min-h-[80vh] flex-col mx-auto px-2 xs:px-0 lg:justify-between justify-center md:container">
-            <div className="flex flex-col lg:px-0">
-              <BackLink year="2024" kind="schedule" />
-
-              {recordingTitle.rating > 0.5 && (
```

Comment:
idk what was the reason to show video based on rating, so I removed this, cc @saihaj

Reply:
e.g. mentioned video in PR was not shown due to rating 0.464

```json
{
  target: 'Comparing API Protocols - One Feature at a Time - Uri Goldshtein, The Guild',
  rating: 0.4642857142857143
}
```